assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -637,15 +637,19 @@ const baseActions = {
 			return { error };
 		}
 
-		// Expire new badges for initially configured audiences.
-		yield commonActions.await(
-			dispatch( CORE_USER ).setExpirableItemTimers(
-				configuredAudiences.map( ( slug ) => ( {
-					slug: `${ AUDIENCE_ITEM_NEW_BADGE_SLUG_PREFIX }${ slug }`,
-					expiresInSeconds: 1,
-				} ) )
-			)
-		);
+		if ( configuredAudiences.length > 0 ) {
+			// Expire new badges for initially configured audiences.
+			yield commonActions.await(
+				dispatch( CORE_USER ).setExpirableItemTimers(
+					configuredAudiences.map( ( slug ) => ( {
+						slug: `${ AUDIENCE_ITEM_NEW_BADGE_SLUG_PREFIX }${ slug }`,
+						expiresInSeconds: 1,
+					} ) )
+				)
+			);
+		}
+
+		return {};
```

Comment:
This fixes an error that occurs when attempting to destructure the return value of the action.